mizani/breaks.py

```diff
@@ -118,7 +118,7 @@ def _log_breaks(limits):
             return base ** _min
 
         step = (_max-_min)//n + 1
-        return base ** np.arange(_min, _max+1, step)
+        return base ** np.arange(_min, _max+1, step, dtype=float)
```

Comment:
As the output should be integer breaks, the result should be cast to integers.

That should fix the failing softest.